packages/mantine/src/components/chip/Chip.tsx

```diff
@@ -0,0 +1,6 @@
+import {Chip as MantineChip, type ChipProps as MantineChipProps} from '@mantine/core';
+import React from 'react';
+
+export type ChipProps = Omit<MantineChipProps, 'variant'>;
+
+export const Chip: React.FC<ChipProps> = (props) => <MantineChip {...props} />;
```

Comment:
I think this will need to be a forward ref, otherwise we cannot set a tooltip on it. Also, we only use named imports of react stuff.

```suggestion
import {Chip as MantineChip, type ChipProps as MantineChipProps} from '@mantine/core';
import {forwardRef} from 'react';

export type ChipProps = Omit<MantineChipProps, 'variant'>;

export const Chip = forwardRef<HTMLInputElement, ChipProps>((props, ref) => <MantineChip ref={ref} {...props} />);
```

Reply:
Yes I forgot about ref thanks!